composer.json

```diff
@@ -1,23 +1,20 @@
 {
     "name": "getdkan/dkan-tools",
     "require": {
-        "consolidation/robo": "^2.0",
+        "consolidation/robo": "^3.0.0",
         "pear/archive_tar": "^1.4",
         "aws/aws-sdk-php": "^3.67",
-        "drupal/coder": "^8.3",
-        "phpunit/phpunit": "^8.5"
```

Comment:
Could we go ahead and remove `/phpunit.xml` also?

Reply:
Good point. Done.